PerformanceTests/Sources/PerformanceTestRunner/Core/Model/Dimension.swift

```diff
@@ -0,0 +1,11 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+struct Dimension: Codable {
+    let name: String  // The name of the dimension. MUST comply with CloudWatch constraints on dimension name.
+    let value: String // The value of the dimension. MUST comply with CloudWatch constraints on dimension value.
```

Comment:
These may as well be doc comments (three slashes on line before property) so that they show in tooltip help

Also, the same sort of comments are appropriate on `PerformanceTestReport` and `PerformanceTestResult` which also form the test report structure.